website/docusaurus.config.js

```diff
@@ -102,29 +102,18 @@ module.exports = {
 	plugins: [
 		require.resolve('./src/plugins/studio-route.cjs'),
 
```

Comment:
(studio cassé)

![image](https://user-images.githubusercontent.com/1730702/204335115-61739069-197f-41e7-a2b4-ac20815f0144.png)

Reply:
strange bug only in prod https://github.com/betagouv/publicodes/pull/299#discussion_r1034002736

Reply:
temporary fix with a `Array.from(new Set())`